packages/react-components/src/DocsComponents/FluentDocsDocsStory.stories.tsx

```diff
@@ -0,0 +1,169 @@
+  SandpackCodeEditor,
+  SandpackPreview,
+  SandpackProvider,
+  SandpackThemeProvider,
+  useSandpack,
+} from '@codesandbox/sandpack-react';
+import '@codesandbox/sandpack-react/dist/index.css';
+import * as dedent from 'dedent';
+import newGithubIssueUrl from 'new-github-issue-url';
+import * as packageJson from '../../package.json';
+import bugReportTemplate from '../../../../.github/ISSUE_TEMPLATE/bug_report_v9.md';
+
+const handleReportBug = (componentName: string) => async (e: React.MouseEvent) => {
+  // create codesandbox
+  let codesandboxUrl = '';
+  const openInCSButton = e.currentTarget.nextElementSibling;
+  const codesandboxForm = openInCSButton?.getElementsByTagName('form').item(0);
+  if (codesandboxForm) {
+    const codesandboxFormData = new FormData(codesandboxForm);
+    codesandboxFormData.append('json', '1');
+    const codesandboxResponse = await fetch(codesandboxForm.action, {
+      method: 'post',
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      body: new URLSearchParams(codesandboxFormData),
+    });
+
+    const sandboxId = (await codesandboxResponse.json()).sandbox_id;
+    codesandboxUrl = `https://codesandbox.io/s/${sandboxId}?file=/example.tsx`;
+  }
+
+  // open GitHub
+  const newIssueUrl = newGithubIssueUrl({
+    repoUrl: packageJson.repository.url,
+    title: '',
+    labels: ['Type: Bug :bug:', 'Fluent UI vNext', componentName ? `Component: ${componentName}` : ''],
+    template: `bug_report_v9_${componentName}_permit.md`,
+    body: bugReportTemplate
+      .replace('[package-versions]', packageJson.version)
+      .replace('[browser-and-os-versions]', navigator.userAgent)
+      .replace('[link-to-codesandbox]', codesandboxUrl),
+  });
+  window.open(newIssueUrl, '_blank');
+};
+
+const LivePreview: React.FunctionComponent<DocsStoryProps & { componentName: string }> = ({ id, componentName }) => {
+  const [renderInline, setRenderInline] = React.useState(true);
+  const sandpack = useSandpack();
+
+  React.useEffect(() => {
+    if (id) {
+      const anchor = document.getElementById(`anchor--${id}`);
+      const iframe = anchor?.getElementsByTagName('iframe')?.item(0);
+
+      // toggle iframe/inline rendering
+      if (iframe) {
+        iframe.style.display = 'none';
+
+        sandpack.listen(msg => {
+          if (msg.type === 'start') {
+            iframe.style.display = 'block';
+            setRenderInline(false);
+          }
+        });
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <SandpackPreview
+      actionsChildren={
+        <>
+          <button
+            type="button"
+            className="sp-button"
+            style={{ padding: 'var(--sp-space-1) var(--sp-space-3)' }}
+            onClick={handleReportBug(componentName)}
+          >
+            Report bug
+          </button>
+        </>
+      }
+    >
+      <>{renderInline && <Story id={id} />}</>
```

Comment:
nit: Unnecessary fragment
```suggestion
      {renderInline && <Story id={id} />}
```

Reply:
I've added the fragment to deal with incompatibility of types:

> Type 'false | Element' is not assignable to type 'Element | undefined'.